### DISCUSSION TITLE
Add a release build for C++

### DISCUSSION THREAD

**Discussion by km-nisei** [Redot-Engine/redot-proposals#62] [Ideas]
It makes no sense to have the engine written almost entirely in C++ and only have support for C# and GDscript. I've heard there are unofficial Godot builds out there, but I think an official build for Redot could be a major hook to get more game devs to use the engine. It could potentially turn into a major boon that really gets the project off the ground, in my opinion.

**Comment by Spartan322** [comment]
See https://github.com/Redot-Engine/redot-cpp

**Comment by Arctis-Fireblight** [comment]
This is now covered by Redot LTS Jenova Edition.
